vc_zoom/indico_vc_zoom/forms.py

```diff
@@ -0,0 +1,100 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2020 CERN and ENEA
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from __future__ import unicode_literals
+from flask import session
+
+from wtforms.fields.core import BooleanField
+from wtforms.fields.simple import TextAreaField
+from wtforms.validators import DataRequired, ValidationError
+
+from indico.modules.vc.forms import VCRoomAttachFormBase, VCRoomFormBase
+from indico.util.user import principal_from_identifier
+from indico.web.forms.base import generated_data
+from indico.web.forms.fields import IndicoRadioField, PrincipalField
+from indico.web.forms.validators import HiddenUnless
+from indico.web.forms.widgets import SwitchWidget
+
+from indico_vc_zoom import _
+
+
+class VCRoomAttachForm(VCRoomAttachFormBase):
+    password_visibility = IndicoRadioField(_("Password visibility"),
+                                           description=_("Who should be able to know this meeting's password"),
+                                           orientation='horizontal',
+                                           choices=[
+                                               ('everyone', _('Everyone')),
+                                               ('logged_in', _('Logged-in users')),
+                                               ('no_one', _("No one"))])
+
+
+class VCRoomForm(VCRoomFormBase):
+    """Contains all information concerning a Zoom booking."""
+
+    advanced_fields = {'mute_audio', 'mute_host_video', 'mute_participant_video'} | VCRoomFormBase.advanced_fields
```

Comment:
```suggestion
    advanced_fields = VCRoomFormBase.advanced_fields | {'mute_audio', 'mute_host_video', 'mute_participant_video'}
```

(dunno, feels almost a bit like a yoda condition to have it the ohter way round)

Reply:
It's not a condition, though. Seems to me like a matter of personal preference.